content/post/Android_Root101.en.md

```diff
@@ -39,26 +42,28 @@ Place this file inside the folder created after extraction.
 
 * Now open up the command prompt on Windows with **_adminitrator priviliges_**.
 
-    _(To do this press the Windows key on your keyboard and search 'cmd' right click when command prompt comes up and chose 'Run as Administrator')_
+    _(To do that, press the `Windows key` on your keyboard and search 'cmd', right click when command prompt comes up and chose 'Run as Administrator')_
 
 * Now go the directory where the files you downloaded have been stored.
     if you don't know how to use Command Prompt
 
 > `cd <dirname>` is use to go up a directory
 
-> `cd..` is used to go down one directory
+> `cd ..` is used to go down one directory
 
+Enable Developer Options on your phone and enable USB-Debugging. To know more [Google Search](https://www.google.com)
 
-Finally pick your phone up and go to the fastboot mode, two methods have been listed below use any:
+Finally, enter fastboot mode on your phone. How-to methods have been listed below(Use any one of the following):
 
-    1. Reboot to Bootloader ->(Hold Power + Volume down)
+1. Reboot to Bootloader ->(Hold Power + Volume down)    {*_For Xiaomi Devices_*}
 
-    2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings
+2. Connect your phone to the PC after enabling the ADB debugging mode in the developer settings
```

Comment:
Did you explain how to do that?
Again, the same thing.
If I have to go search how to do things you have mentioned, Why would I read your article?
I'd go find a better one